assets/collage.css

```diff
@@ -1,4 +1,5 @@
 .collage-wrapper-title {
```

Comment:
Could probably be standardized and simply use something like `.title` in the markup instead of one-off `.collage-wrapper-title`. Can look into this with the other ["title" issues](https://github.com/Shopify/dawn/pull/1006#issuecomment-996892693) you mentioned.

Reply:
Created an issue a while back for it: https://github.com/Shopify/dawn/issues/286

I want to get rid all of the wrapper classes except maybe the one that handles titles with link.  They are a real mess.